src/trupti/BankSystem.java

```diff
@@ -0,0 +1,78 @@
+package trupti;
+/*Assignment 9 : 2nd April 2021
+Create a Banking System which has the following functionality.
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+public class BankSystem {
+	 int debitCnt=0;
+	 int creditCnt=0;
+	 int balance=10000,balanceCnt=0;
+	 static int totalDebitCount=0;
+	 static int totalCreditCount=0;
+	 static int totalBalanceCount=0;
+	
+	 void debitAmt(int debitAmount) {
+				
+		 balance=balance-debitAmount;
```

Comment:
For debit operation, you should write if condition to check whether balance is sufficient or not.